tutorials/process_per_core_translation/cpu_process_per_core_translation.py

```diff
@@ -0,0 +1,129 @@
+# Describtion: This script is used for CPU Multi-instance Translate, which process per core translation.
+# It can greatly speedup translate perefomance.
+# FileName: cpu_process_per_core_translation.py
```

Comment:
filename and version seem unnecessary. The usage would be best as part of the argparse description `argparse.ArgumentParser(description=...`.

Reply:
Done. Thanks